app/src/userpages/tests/components/KeyField/KeyField.test.jsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import assert from 'assert-diff'
+import sinon from 'sinon'
+
+import KeyField from '$userpages/components/KeyField'
+import TextInput from '$shared/components/TextInput'
+import DropdownActions from '$shared/components/DropdownActions'
+import KeyFieldEditor from '$userpages/components/KeyField/KeyFieldEditor'
+
+describe('KeyField', () => {
+    let sandbox
```

Comment:
It should work just fine as a `const` that's restored after each example. No `beforeEach` required, i.e.
```js
const sandbox = sinon.createSandbox()

afterEach(() => {
    sandbox.restore()
})